disco_aws_automation/disco_vpc.py

```diff
@@ -726,222 +466,40 @@ def _destroy_vpc(self):
         # save function and parameters so we can delete dhcp_options after vpc. We do this becase botos
         # get_all_dhcp_options does not support filter. Because we cannot easily find the default dhcp
         # options, re-assigning default dhcp option is not trivial.
-        delete_dhcp_options = self.vpc.connection.delete_dhcp_options
-        dhcp_options_id = self.vpc.dhcp_options_id
+        # delete_dhcp_options = self.vpc.connection.delete_dhcp_options
+        dhcp_options_id = self.vpc['DhcpOptionsId']
 
-        delete_status = keep_trying(30, self.vpc.delete)
+        self.boto3_ec2.delete_vpc(VpcId=self.get_vpc_id())
+        # delete_status = keep_trying(30, self.vpc.delete)
         self.vpc = None
 
-        if not delete_dhcp_options(dhcp_options_id):
-            logging.warning("failed to delete dhcp options (%s)", dhcp_options_id)
+        self.boto3_ec2.delete_dhcp_options(DhcpOptionsId=dhcp_options_id)
+        # if not delete_dhcp_options(dhcp_options_id):
+        #    logging.warning("failed to delete dhcp options (%s)", dhcp_options_id)
 
-        return delete_status
+        # return delete_status
```

Comment:
Yeah, the `destroy_vpc_command` method in `disco_vpc_ui.py` is not making use of the returned value. I'll delete that.